### PR TITLE
enh(UI): Add props to track user when they click on component

### PR DESCRIPTION
**Pull request by Jordan91**
Add props testId to track user in submenuItem

### CODE REVIEW

File: packages/centreon-ui/src/Listing/ActionBar/index.tsx [hunk 1]
```diff
@@ -93,9 +93,6 @@ const ListingActionBar = ({
       {paginated && (
         <StyledPagination
           ActionsComponent={PaginationActions}
-          SelectProps={{
-            native: true,
-          }}
```
Comment (Thebarda) [inline packages/centreon-ui/src/Listing/ActionBar/index.tsx:98]: Why this ?

Comment (Jordan91) [reply packages/centreon-ui/src/Listing/ActionBar/index.tsx:98]: A bad merge

File: packages/centreon-ui/src/Submenu/SubmenuHeader/SubmenuItem/index.tsx [hunk 2]
```diff
@@ -67,12 +67,16 @@ interface Props {
...
+  testIdCount?: string;
+  testIdTitle?: string;
 }
 
 const SubmenuItem = ({
   dotColored,
   submenuTitle,
   submenuCount,
+  testIdTitle,
+  testIdCount,
```
[the remark this answers is not in the document]
Comment (Jordan91) [reply packages/centreon-ui/src/Submenu/SubmenuHeader/SubmenuItem/index.tsx:79]: Ok As you suggest last friday

[the remark this answers is not in the document]
Comment (Jordan91) [reply packages/centreon-ui/src/Submenu/SubmenuHeader/SubmenuItem/index.tsx]: I think after review with Bruno, this component is too generic to handle type of resource who used (host or service) are you ok ?

**Review by Thebarda — APPROVED** [review]

**Review by Thebarda — DISMISSED** [review]

**Review by Thebarda — APPROVED** [review]

**Review by Thebarda — APPROVED** [review]

**Review by Thebarda — APPROVED** [review]

**Review by bdauria — APPROVED** [review]

**Pull request merged**